### ISSUE TITLE
Modify the request/limit for all containers that are in the Sidecar

### ISSUE THREAD

**Issue by Juanfree** [okteto/okteto#1294]
**Is your feature request related to a problem? Please describe.**
On the cluster that I am working, we have limitranges configured and the min request is 256Mi.

When I try to make the okteto up I receive and error, because I have the issue as I said with the limitranges.

**Describe the solution you'd like**
We able to inject some configuration in order to increase/modify the request/limit.

**Comment by pchico83** [comment]
We could provide a section to fine tune the init container. Something like this:
```
initContainer:
   image: xxx
   resources:
      requests:
          cpu: xxx
          memory: xxx
      limits:
          cpu: xxx
          memory: xxx

```

**Comment by Juanfree** [comment]
That's will be on the okteto.yml?

For me sounds really nice ;)

**Comment by boufleur** [comment]
This can also fix the issue where you want to download the image through an on premises image repository proxy.

**Comment by Juanfree** [comment]
As far as I understand the image property on the okteto.yml will no be mandatory, right? I mean, if is not defined will take the default?

**Comment by pchico83** [comment]
@Juanfree yes, these fields will be optional, with default values to the current behavior

**Comment by Juanfree** [comment]
Hi,

When we are going to have this feature live?

Thanks a lot for the effort and OFC for the product! It's really amazing!

**Comment by pchico83** [comment]
@Juanfree the PR is already approved and ready to merge and we want to make an Okteto CLI release later today including it

**Comment by pchico83** [comment]
@Juanfree this is now available in Okteto CLI 1.11.2.
Docs are available here:
https://okteto.com/docs/reference/manifest/index.html#initcontainer-object-optional

**Comment by Juanfree** [comment]
ok! thanks a lot!
